sensorimotor/experiments/capacity/params/reasonable/2048.yaml

```diff
@@ -1,5 +1,7 @@
 n: 2048
```

Comment:
Do you want two files, one for online=True and another for online=False? I suppose we might not even want online=False in the near future, but for now I'd like to add this.